1.0-specification.norg

```diff
@@ -1235,8 +1235,8 @@ version: 1.0
    serve different use cases.
 
    The content of attached modifier extensions consists of a set of references to many
-   {*** attributes}. These attributes are delimited by the {* contextual `|` delimiter}.
```

Comment:
The attributes section was removed in [this commit](https://github.com/nvim-neorg/norg-specs/commit/51aab3dfd3bdf1d7ebdd2648d2e418ddd8d9ce2c#diff-e5f965a07482237b76a47140269c90cf4d0e3ceaabbcf96c7a047c59a98713a5L340), not sure if intentional or not